### PR TITLE
Added roadmap doc

### PR DESCRIPTION
**Pull request by wesleymonte**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Roadmap
+
+The SAPS is currently composed of four main services: `Archiver`, `Catalog`, `Dispatcher` and `Scheduler`. In addition, saps uses a job execution service called `Arrebol`. Given the context, below we have the correct deployment sequence for Saps components.
```
Comment (raonismaneoto) [inline docs/README.md]: SAPS is currently composed... no need for 'THE'

Comment (raonismaneoto) [reply docs/README.md]: Saps uses a job execution service, but it has to be arrebol? or can it be whichever the user wants? If thats the case u could say as arrebol for example.

**Review by raonismaneoto — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (wesleymonte) [reply docs/README.md]: Yes, the execution service can be other than Arrebol. I added in the doc.

**Review by raonismaneoto — APPROVED** [review]

**Review by thiagoyeds — APPROVED** [review]
Lgtm

**Pull request closed without merging**